templates/model/README.md

```diff
@@ -46,25 +46,31 @@ something isn't right. You can use
     `Upgrade()` method so that it stored the contents of the old field in the
     newly renamed field.
 - In `templates/model/decode/decode.go`, add a new entry to the end of
-  `apiVersions`. See the instructions and examples there.
+  `apiVersions`(If new api version
+  is still WIP and is not ready to get released, mark the `unreleasd` filed to be true.).
```

Comment:
Spelling and grammar nits:

unreleasd -> unreleased

`true.).` -> `true).`

There should be a space before each `(`.

Reply:
done